.gitignore

```diff
@@ -100,8 +100,8 @@ config.yaml
 # Vscode
 .vscode
 
-# Chromedriver
-chromedriver/
+# Chromedriver / Webdriver Manager
+.wdm/
```

Comment:
Doesn't WDM create this directory inside a user's home anyways?

Reply:
If the WDM_LOCAL environment variable is set, it caches the file in the project directory. I use this to get around user ID issues when building the docker container.